syftlib/src/main/java/org/openmined/kotlinsyft/Socket.kt

```diff
@@ -0,0 +1,18 @@
+package org.openmined.kotlinsyft
+
+import java.net.Socket
+
+
+class Socket(private val url: String, private val workerId: String, keepAliveTimeout: Int = 20000) {
+
+    val socket = Socket(url, 888)
```

Comment:
Should the port be passed as a param too?

Reply:
This is still just for representation. I still need to implement a signaling client. This would be implemented in a different issue so that people can start on it directly using webRTC